setup.py

```diff
@@ -0,0 +1,48 @@
+import setuptools
+
+with open("readme.md", "r") as f:
+    long_description = f.read()
+
+setuptools.setup(
+    name="open-mcr",
+    url = "https://github.com/gutow/open-mcr",
+    version="1.3.0dev",
+    description="Graphical tool for optical mark recognition for test scoring.",
+    long_description=long_description,
+    long_description_content_type="text/markdown",
+    author="Ian Sanders",
+    author_email="iansan5653@github.com",
```

Comment:
I'd just prefer to use my personal email for this (I might not work at GitHub forever 😄):
```suggestion
    author_email="iansan5653@gmail.com",
```